src/navigators/TabBar.tsx

```diff
@@ -260,57 +262,35 @@ const defaultStyles = createStyles('TabBar', (theme) => ({
   },
   labelActiveText: {
     alignSelf: 'center',
+    color: theme.colors.onSurface,
   },
-  labelActiveText0: {},
-  labelActiveText1: {},
-  labelActiveText2: {},
   labelInactiveText: {
     alignSelf: 'center',
+    color: theme.colors.onSurfaceDisabled,
   },
-  labelInactive0Text: {},
-  labelInactive1Text: {},
-  labelInactive2Text: {},
   activeIndicatorView: {
     position: 'absolute',
-    top: -6,
     height: 2 * 3,
     zIndex: 1,
-  },
-  activeIndicator0View: {
-    backgroundColor: theme.colors.primary,
-  },
-  activeIndicator1View: {
-    backgroundColor: theme.colors.secondary,
-  },
-  activeIndicator2View: {
-    backgroundColor: theme.colors.tertiary,
+    backgroundColor: theme.colors.scrim,
   },
   tabActiveView: {
     height: '100%',
     paddingTop: 2,
-  },
-  tabActive0View: {
-    backgroundColor: theme.colors.primaryContainer,
-  },
-  tabActive1View: {
-    backgroundColor: theme.colors.secondaryContainer,
-  },
-  tabActive2View: {
-    backgroundColor: theme.colors.tertiaryContainer,
+    backgroundColor: theme.colors.background,
   },
   tabInactiveView: {
     height: '100%',
     paddingTop: 2,
+    backgroundColor: theme.colors.background,
```

Comment:
Same here as comment above, it seems like (from the removed entries below) this used to be using `surface` and you're changing it to use `background`.  Is that correct, and if so are these intentional changes?